### PR TITLE
Generate comments data with random categories for variation

### PR DESCRIPTION
**Pull request by javierjulio**
Since we support comments we should have data to begin with when loading up a test app to try out the admin. I used categories since we only have a couple, this way we get many comments for one of them and can active pagination. We've had some issues submitted on that lately and would be good to have this in place for testing, also for updates to come in v2 (#3862).

### CODE REVIEW

File: spec/support/rails_template_with_data.rb [hunk 1]
```diff
@@ -313,6 +313,15 @@
                 author: user,
                 starred: true
   end
+
+  800.times do |i|
+    ActiveAdmin::Comment.create!(
+      namespace: :admin,
+      author: AdminUser.first,
+      body: "Test comment #{i}",
+      resource: categories[rand(categories.size - 1)],
```
Comment (Fivell) [inline spec/support/rails_template_with_data.rb]: @javierjulio , since this is array we can simplify it using ```categories.sample```

Comment (javierjulio) [inline spec/support/rails_template_with_data.rb]: Agreed, that's much better! I didn't know about `sample`.

Comment (javierjulio) [inline spec/support/rails_template_with_data.rb]: @Fivell thanks, I've updated this!

**Comment by javierjulio** [comment]
@Fivell thanks! I'll merge once build passes.

**Pull request merged**